dist/src/main/java/io/camunda/zeebe/gateway/StandaloneGateway.java

```diff
@@ -40,7 +42,8 @@
       "io.camunda.zeebe.gateway",
       "io.camunda.zeebe.shared",
       "io.camunda.zeebe.util.liveness"
-    })
+    },
+    exclude = {GraphQLJavaToolsAutoConfiguration.class, GraphQLAnnotationsAutoConfiguration.class})
```

Comment:
Should be added to `application.properties` in the `dist` module. The same applies to the other application classes.

Reply:
indeed, all applications except `Tasklist`

Reply:
@romansmirnov done in https://github.com/camunda/zeebe/pull/17860/commits/aaafb767624590e3ab9abe9f5f97969c83dfb1c4
For Tasklist I had to override `spring.autoconfigure.exclude` to re-enable `graphql.kickstart.autoconfigure.annotations.GraphQLAnnotationsAutoConfiguration`, is there other way to do so?